src/Servers/Kestrel/Core/src/Internal/KestrelServerImpl.cs

```diff
@@ -325,10 +325,12 @@ private async Task BindAsync(CancellationToken cancellationToken)
             }
         }
 
-        private static void TriggerRebind(object state)
+        private static void TriggerRebind(object? state)
```

Comment:
We always register this callback with a state, so a Debug.Assert should be sufficient.